ledger/eval.go

```diff
@@ -1130,7 +1130,7 @@ func (eval *BlockEvaluator) GenerateBlock() (*ValidatedBlock, error) {
 			"unknown consensus version: %s", eval.block.BlockHeader.CurrentProtocol)
 	}
 	eval.state = makeRoundCowState(
-		eval.state, eval.block.BlockHeader, proto, eval.prevHeader.TimeStamp,
+		eval.state, eval.block.BlockHeader, proto, eval.prevHeader.TimeStamp, eval.state.mods.Totals,
```

Comment:
use the Totals from the previous generated block into the next one.

Reply:
looks like `eval.state.mods` has current data and not previous one:

```
// added some txns from a client
err := pool.pendingBlockEvaluator.TransactionGroup(txgroupad)
...
// generated block from current data
lvb, gerr := pool.pendingBlockEvaluator.GenerateBlock()
...
// saved
pool.assemblyResults.blk = lvb
```

Reply:
maybe I'm missing something - we use the current "finalized" totals in one round as the basis for the subsequent round. The `roundCowState.prevTotals` contains the previous round totals, and it being set up in `makeRoundCowState`. The `Totals` is calculated in `CalculateTotals`

Reply:
So is it creating a new `cow` for subsequent round and `eval.state` is `rowCowBase`? if so then looks correct.